site/src/pages/WorkspaceSettingsPage/WorkspaceSchedulePage/WorkspaceScheduleForm.tsx

```diff
@@ -311,14 +315,14 @@ export const WorkspaceScheduleForm: FC<
           <Stack direction="row">
             <TextField
               {...formHelpers("startTime")}
-              disabled={isLoading || !form.values.autostartEnabled}
```

Comment:
I am not sure why we were looking at the form values here...I assume this was a typo.

Reply:
I think it's because that's where the state of the form is. So if someone turns on autostart, this is the place to check its value.

Reply:
Thanks for catching me here - I've fixed the attribute and added some test cases.
Down the road, we might want to reconsider the prop architecture of the form as it accepts `initialValues`, and one could try to disable `autostart` or  `autostop` using this prop, e.g. :
```
initialValues={{
  autoStartEnabled: false
  autoStopEnabled: false
}}
```
But this wouldn't affect the top toggle of each feature; only the secondary fields associated with them. So it's a bit of a bizarre API, but I'd rather not touch it in this PR.